src/main/java/Model/Group.java

```diff
@@ -1,10 +1,20 @@
 package Model;
 
+import java.util.ArrayList;
+
 public class Group {
     private String id;
     private String name;
     private String groupLink;
+    private ArrayList<String> notificationsList;
+    private ArrayList<String> pendingRequestsList;
+    private ArrayList<String> usersList;
 
+    public Group(){
+        this.id = null;
+        this.name = null;
+        this.groupLink = null;
```

Comment:
Do we really required to initialize them with `null`?

Reply:
Sorry. That was redundant. Fixed it.